templates/base.html

```diff
@@ -21,7 +21,7 @@
         <link type="application/atom+xml" rel="alternate" href="/{{config.feed_filename}}" title="{{config.title}}">
         {% endif %}
     <!-- mathjax support -->
-    <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
+    <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
```

Comment:
```suggestion
```
According to [upstream MathJax](https://docs.mathjax.org/en/latest/web/start.html#browser-compatibility), this is only needed for IE11 support.
Let's just remove it entirely.